datahub-web-react/src/app/permissions/policy/policyUtils.ts

```diff
@@ -126,3 +127,150 @@ export const setFieldValues = (
     }
     return { ...filter, criteria: [...restCriteria, createCriterion(resourceFieldType, fieldValues)] };
 };
+
+
+/**
+ * Add an entry to the ListPolicies cache.
+ */
+export const addToListPoliciesCache = (client, newPolicy, pageSize, query) => {
```

Comment:
can we create a unit test for these functions?

Reply:
Sure I will try to add test case here